### DISCUSSION TITLE
Wann gebraucht man das?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2558] [General]
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/38959acc-b75c-4748-b9e9-7dc95c926373)
Kann man dann über Deinen Server das System nutzen und nicht über ioBroker.net?

**Comment by Zefau** [comment]
(Ich hab mal den Screenshot entfernt)

Das wird mit v4.0 kommen, dass jarvis über das Internet nutzbar ist. Habe hier bereits einige Vorarbeit geleistet. Die Einstellungen werden ich bis dahin erstmal aus der v3.2 entfernen.

**Comment by mcuiobroker** [comment]
Solltest dann auch die automatische Umleitung auf die Seite rausnehmen, bin nicht der einzige der es testet?

**Comment by Zefau** [comment]
Wann kommt da eine Umleitung?

**Comment by mcuiobroker** [comment]
Ich bin ja auf www.zefau.net gegangen und nicht direkt auf die darunter liegende Seite.

Schreib doch bitte in der Adapter-Konfig dazu, dass es erst mit v4.0 kommt
